src/lib/provable/group.ts

```diff
@@ -101,67 +106,49 @@ class Group {
         return fromProjective(g_proj);
       }
     } else {
-      const { x: x1, y: y1 } = this;
```

Comment:
this code was moved to `native-group.ts` to reuse there. it was also rewritten a bit and a bug fixed